1224-user_administration/administrator_opens_user_administration.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Administrator opens user administration
+
+    @359
+    Scenario Outline: Administrator opens User Administration
+        Given that the user is logged in as Administrator
+        When they click the menu item Users
+        Then they see the User Administration page
+        And they see the Section "<Role>"
+        And each Section "<Role>" lists all users affiliated with their institution with role "<Role>"
+        And they see a Button "<Button>" to assign the Role "<Role>" to another user
```

Comment:
Når man bruker "<xxx>" skal det bli en fullstendig setning når man setter inn verdiene fra Examples.
"And each Section "Adminstrator" lists all users(...)" blir rart.

```suggestion
        And they see Section "<Section>"
        And they see Section "<Section>" with users affiliated with their institution with role "<Role>"
        And they see a Button "<Button>"
```

og utvide Examples med:
```
        Examples:
            | Section             | Role                       | Button            |
            | Administrators | Adminstrator          |  Add Administrator |
```